dev-support/pysubmarine/openapi.json

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
I'm wondering do we need to maintain this openapi.json file since we have the automation script to generate it?

Reply:
should we remove it?
Maybe someone will want to check `openapi.json` structure, and they don't want to build the project by themself. (currently, if we want to generate `openapi.json`, we need to build project firstly)

Reply:
Let's keep it then.